### ISSUE TITLE
STIX export requires newer STIX Python libraries

### ISSUE THREAD

**Issue by RichieB2B** [MISP/MISP#3261]
Because of #3260 the STIX Python libraries MISP uses now need to be Python 3 compatible. Version 1.1.1.4 is clearly not:
```
Traceback (most recent call last):
  File "/tmp/stixtest.py", line 2, in <module>
    import stix
  File "/usr/local/lib/python3.5/dist-packages/stix/__init__.py", line 12, in <module>
    from .base import Entity, EntityList, TypedList, BaseCoreComponent  # noqa
  File "/usr/local/lib/python3.5/dist-packages/stix/base.py", line 8, in <module>
    import StringIO
ImportError: No module named 'StringIO'
```
So the git submodule and [stixDiagnostics()](https://github.com/MISP/MISP/blob/2.4/app/Model/Server.php#L3159) needs to be updated.

**Comment by RichieB2B** [comment]
@chrisr3d what stix/cybox/mixbox versions did you test your code with?

**Comment by RichieB2B** [comment]
Ok, I just tried the STIX export on https://misp.circl.lu/ and it gives me an `HTTP 500`. I think we can officially call the STIX export of MISP broken until @chrisr3d documents the stix/cybox/mixbox versions required for his code to work.

**Comment by iglocska** [comment]
He was working on that, will update all libraries when the update docs are in place (think it might be already, but in a meeting ;))

**Comment by iglocska** [comment]
https://github.com/MISP/MISP/blob/2.4/INSTALL/UPDATE.txt#L36

Looks like he did it 4 days ago - I fail for not updating it on the circl server

**Comment by chrisr3d** [comment]
If I did not miss any other file to update, we just need to change the Diagnostic tool

**Comment by iglocska** [comment]
yep, saw it, I was slow with the update of the actual server ;) True for the diagnostic tool, completely forgot about that.

**Comment by RichieB2B** [comment]
I just installed a fresh Ubuntu 18.04 LTS Server image and followed the MISP install instructions. These still tell me to run `python setup.py install` instead of `python3 setup.py install` for the stix/cybox/mixbox modules. So I did this all manually. Running `python3 app/files/scripts/stixtest.py` now gives:
```
{"success": 1, "stix": "1.2.0.6", "cybox": "2.1.0.18.dev0", "mixbox": "1.0.3"}
```
However, downloading an event in STIX format still gives an `HTTP 500` because of pymisp not being installed. Finally, after a `pip3 install pymisp` STIX export works again.

Conclusion: the UPDATE.txt and INSTALL.txt instructions need to be updated to reflect the change from python2 to python3 for STIX export.
